### PR TITLE
Fix virtio_scsi_cdrom windows installation issue

### PR DESCRIPTION
**Pull request by qingwangrh**
1. Add auto send key action of ovmf windows installation
2. Reduce disks number of multi disks windows installation

ID:1874693
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2016,RHEL.8.4.0 --driveformat=virtio_blk,virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35,i440fx   --clone=no --firmware=ovmf --nrepeat=1


(1/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1031.62 s)
 (2/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (583.04 s)
 (3/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1063.45 s)
 (4/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (560.42 s)

 python ConfigTest.py --testcase=virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=RHEL.8.4.0,Win2016 --driveformat=virtio_blk,virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35,i440fx   --clone=no


(1/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (1211.84 s)
 (2/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1202.78 s)
 (3/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (382.82 s)
 (4/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (555.87 s)
 (5/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (1686.42 s)
 (6/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1327.65 s)
 (7/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (614.62 s)
 (8/8) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (566.89 s)

**Comment by qingwangrh** [comment]
@YongxueHong @zhenyzha @menli820 Please help to review

**Comment by zhenyzha** [comment]
It seems that only the windows part of the code has been changed here. Since neither aarch64 nor Power PC supports windows testing, please ignore me.

**Comment by menli820** [comment]
@qingwangrh  the reason why need to reduce disk is for https://bugzilla.redhat.com/show_bug.cgi?id=1714485#c13 right？

**Comment by menli820** [comment]
Add @peixiu  for the scsi part, thanks

**Comment by menli820** [comment]
> @qingwangrh the reason why need to reduce disk is for https://bugzilla.redhat.com/show_bug.cgi?id=1714485#c13 right？

yes, already confirm with qinwang on rc

**Comment by menli820** [comment]
LGTM

blk+seabios:
 (1/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (1177.21 s)
 (2/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1048.98 s)
 (3/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (695.91 s)
 (4/4) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (6049.76 s)

blk+ovmf:
 (1/1) Host_RHEL.m8.u5.product_av.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (971.07 s)

File: qemu/tests/cfg/virtio_scsi_cdrom.cfg [hunk 1]
```diff
@@ -68,12 +73,39 @@
                     image_name_stg22 = images/storage22
                     image_name_stg23 = images/storage23
                     image_name_stg24 = images/storage24
-                    image_name_stg25 = images/storage25
                     image_size_equal = 1G
```
Comment (peixiu) [inline qemu/tests/cfg/virtio_scsi_cdrom.cfg:76]: Hi qinwang, if the param image_size_equal = 1G is unused?

Comment (peixiu) [reply qemu/tests/cfg/virtio_scsi_cdrom.cfg:76]: (1/1) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1051.41 s)

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install: PASS (690.56 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (898.38 s)

**Comment by peixiu** [comment]
LGTM～

 (1/1) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.virtio_scsi_cdrom.with_installation.cdrom.extra_cdrom_ks.aio_threads.multi_disk_install.q35: PASS (1070.03 s)

Acked-by: Peixiu Hou phou@redhat.com

**Review by peixiu — APPROVED** [review]

**Review by YongxueHong — APPROVED** [review]
LGTM

**Pull request merged**